start-up/setting-up-reaction-roles.md

```diff
@@ -13,43 +13,37 @@ Before beginning, make sure of the following things:
 
 ## Creating a group
 
-A group is a collection of emoji to role pairs and settings. Groups are independent for each other. After setting the appropriate pairs, you can apply the group to messages so users can start interacting with the bot and gain roles.
+A group is a collection of emoji to role pairs and settings. Groups are independent for each other. After setting the appropriate pairs, you can apply the group to messages so members can use the reactions to manage their roles.
 
 To create a group, use this command:
 
 ```
-/reaction-roles create <name>
+/config reaction-roles <group>
 ```
 
-Replace `<name>` with a short, memorable name that you will remember later for every reaction roles operation.
+Replace `<group>` with a short, memorable name that you will remember later for every reaction roles operation.
 
-{% hint style="info" %}
-Screenshots in this page will show old, message-based commands. Type `/reaction-roles` while using any command now. We are working to replace the outdated illustrations as soon as possible.
-{% endhint %}
-
-![Creation of a group named "pets"](<../.gitbook/assets/immagine (9).png>)
+![](<../.gitbook/assets/immagine (9).png>)
 
 Group names can contain only alphanumeric characters, underscores and dashes. They must be unique and no longer than 32 characters.
 
 ## Matching the pairs together
 
-Now that you have a group made, you need to configure what emoji users need to react to get roles.
+Now that you have a group made, you need to configure which emojis users need to react with to get roles.
 
-You do that with the `setroles` command in this fashion:
+You do that with the `set` command in this fashion:
 
 ```
-/reaction-roles setroles <group>
+/reaction-roles set <group> <emoji> <role>
 ```
 
-The argument, `<group>` is the name of your group. Using this command will begin a guided process to set your roles. First, you will be asked to input a role (with a name, ID or mention), then react to the reply message to tie it to an emoji.
-
-![Step 1: the role (yellow), step 2: the reaction (green)](<../.gitbook/assets/immagine (10).png>)
-
-Add as many as you want, or stop adding roles with `cancel`
+* `<group>` = the name of the group you set up before.
+* `<emoji>` = the emoji you want to be used
+* `<role>` = the role you want the emoji to be paired to
 
-![](<../.gitbook/assets/immagine (11).png>)
+![](<../.gitbook/assets/immagine (10).png>)
 
-You can use custom emojis as well, but they must be uploaded in the same server as you're setting it up.
+You can use this command as many times as you want to add multiple pairs to your group. You can use custom emojis as well, but they must be uploaded in the same server as you're setting it up.
```

Comment:
I don't understand

Reply:
`/ [...] set group1 emoji1 role1`

`/ [...] set group1 emoji1 role2`

Doing this will give both roles at the press of the reaction. To remove role1 from the emoji you can do `/ [...] set group1 emoji1 role1`